modules/oe_content_person/oe_content_person.post_update.php

```diff
@@ -72,3 +75,56 @@ function oe_content_person_post_update_20003(): void {
     $field_config->save();
   }
 }
+
+/**
+ * Add Description field.
+ */
+function oe_content_person_post_update_30001(): void {
+  $storage = new FileStorage(\Drupal::service('extension.list.module')->getPath('oe_content_person') . '/config/post_updates/30001_add_description_field');
+
+  $configs = [
+    'field_storage_config' => [
+      'field.storage.node.oe_person_description' => 'node.oe_person_description',
+    ],
+    'field_config' => [
+      'field.field.node.oe_person.oe_person_description' => 'node.oe_person.oe_person_description',
+    ],
+  ];
+
+  foreach ($configs as $key => $ids) {
+    $config_storage = \Drupal::entityTypeManager()->getStorage($key);
+    foreach ($ids as $id => $reference) {
+      if (!$config_storage->load($reference)) {
+        $config_data = $storage->read($id);
+        // We are creating the config which means that we are also shipping
+        // it in the install folder so we want to make sure it gets the hash
+        // so Drupal treats it as a shipped config. This means that it gets
+        // exposed to be translated via the locale system as well.
+        $config_data['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config_data));
+        $config_storage->create($config_data)->save();
+      }
+    }
+  }
+
+  // Form display configuration to update.
+  $form_display_values = $storage->read('core.entity_form_display.node.oe_person.default');
+  $form_display_values['_core']['default_config_hash'] = Crypt::hashBase64(serialize($form_display_values));
+  $form_display = EntityFormDisplay::load($form_display_values['id']);
+  if ($form_display) {
+    $updated_form_display = \Drupal::entityTypeManager()
+      ->getStorage($form_display->getEntityTypeId())
+      ->updateFromStorageRecord($form_display, $form_display_values);
+    $updated_form_display->save();
+  }
+
+  // View display configuration to update.
+  $view_display_values = $storage->read('core.entity_view_display.node.oe_person.default');
+  $view_display_values['_core']['default_config_hash'] = Crypt::hashBase64(serialize($view_display_values));
+  $view_display = EntityViewDisplay::load($view_display_values['id']);
+  if ($view_display) {
+    $updated_view_display = \Drupal::entityTypeManager()
+      ->getStorage($view_display->getEntityTypeId())
+      ->updateFromStorageRecord($view_display, $view_display_values);
+    $updated_view_display->save();
+  }
```

Comment:
We should update the view display only in oe_theme where we take care of the rendering.

Reply:
removed.